### PR TITLE
adding unit-test

### PR DESCRIPTION
**Pull request by lukaszszymankiewicz**
siusiak drugi raz

### CODE REVIEW

File: tests/test_src.py [hunk 1]
```diff
@@ -8,31 +8,15 @@ def test_set_prices_sorting_works():
...
-
-    #
-    assert expected == result
-
-def test_set_prices_multiplying_works_with_3():
-    #
-    prices = [10, 30, 20]
-    a = 3
-    expected = [30, 60, 90]
+    # TODO: write me!
```
Comment (MichalLuczak143) [inline tests/test_src.py:21]: kruci

File: tests/test_src.py [hunk 2]
```diff
@@ -8,31 +8,15 @@ def test_set_prices_sorting_works():
     #
     result = set_prices(prices, 1.0)
 
-    #
+    # TODO: correct me!
+
+    # if result_sorted != result:
+    #     raise AssertionError()
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:14]: first comment

File: tests/test_src.py [hunk 3]
```diff
@@ -8,31 +8,15 @@ def test_set_prices_sorting_works():
...
-
-    #
-    assert expected == result
-
-def test_set_prices_multiplying_works_with_3():
-    #
-    prices = [10, 30, 20]
-    a = 3
-    expected = [30, 60, 90]
+    # TODO: write me!
```
Comment (lukaszszymankiewicz) [inline tests/test_src.py:20]: second comment

File: tests/test_src.py
Comment (MichalLuczak143) [inline tests/test_src.py:1]: IMG_1679

**Review by lukaszszymankiewicz — COMMENTED** [review]
this looks like shit

**Pull request closed without merging**